src/views/NotFound.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div id="main">
+    <p>404 Page Not Found</p>
+    <img src="../assets/WalnutImage.jpg" />
+    <button onclick="location.href='myschedule'">
```

Comment:
This is correct HTML, but in Vue, we generally use Vue's [`v-on` directive](https://vuejs.org/v2/guide/#Handling-User-Input) so that we can make use of Vue's methods and stuff.

```
<button v-on:click="location.href='myschedule'">
```
This is usually abbreviated to `@click`:
```
<button @click="location.href='myschedule'">
```

Also, since we're using Vue Router, it's faster to use Vue Router's [programmatic navigation](https://router.vuejs.org/guide/essentials/navigation.html).
```
<button @click="$router.push('/schedule')">
```